spec/javascripts/controllers/schedule/schedule_form_controller_spec.js

```diff
@@ -715,4 +715,43 @@ describe('scheduleFormController', function() {
       });
     });
   });
+
+  describe('saveable should exist in the scope', function() {
+    it('returns true', function() {
+      expect($scope.saveable).toBeDefined();
+    });
+  });
+
+  describe('Validates credential fields', function() {
+    beforeEach(inject(function($compile, miqService) {
+      var angularForm;
+      var element = angular.element(
+        '<form name="angularForm">' +
+        '<input ng-model="scheduleModel.depot_name" name="depot_name" ng-required="scheduleModel.depot_name.length==0" text />' +
+        '<input ng-model="scheduleModel.uri" name="uri" ng-required="scheduleModel.uri.length==0" text />' +
```

Comment:
@himdel ```ng-required=true``` and ```required``` are essentially the same thing.
So in the above example, when ```scheduleModel.uri.length==0``` evaluates to ```true```, it boils down to ```<input required>``` (what you suggested)
Anyway, to favor brevity, I will be changing it to ```<input required>```